sinabs/backend/dynapcnn/chips/dynapcnn.py

```diff
@@ -73,12 +105,30 @@ def set_kill_bits(cls, layer: DynapcnnLayer, config_dict: dict) -> dict:
         return config_dict
 
     @classmethod
-    def get_dynapcnn_layer_config_dict(cls, layer: DynapcnnLayer):
+    def get_dynapcnn_layer_config_dict(
```

Comment:
get_dynapcnn_layer_config_dict and write_dvs_layer_config receive the same parameters besides chip_layer.
Are those functions exposed to users? If so, can we have the parameters in the same order?

Reply:
I am confused about how I read the parameters in a different order. It looks like it is the same order, indeed.